src/tinygp/kernels/quasisep.py

```diff
@@ -643,21 +646,27 @@ class CARMA(Quasisep):
 
     .. code-block:: python
 
-        kernel = CARMA.init(alpha=..., beta=..., sigma=...)
+        kernel = CARMA.init(alpha=..., beta=...)
+
+    .. note::
+        To fit a CARMA model with p > 2, the :func:`from_fpoly` method needs to
+        be used to construct a valid model.
```

Comment:
Maybe we should disallow the `alpha` / `beta` initialization entirely and just support `from_fpoly` instead (perhaps renaming it to `init`)?

As an aside: what does "fpoly" mean?

Reply:
I can see that it is confusing to have two constructors, but I think it might be more intuitive to stick with the standard parameterization? 

The reason for using the `from_fploy` constructor is that for p > 2 not every combination of positive CARMA parameters gives valid solutions. I am thinking to add a new step within the `init` constructor to search for the closest valid solution. Typically valid solutions are scattered in the nominal CARMA space. This step will add a small overhead, but I need to test to see how much. What do you think? Other suggestions?

`fpoly` means factorized polynomial = a product of quadractic equation. Yeah, not very intuitive. I renamed it to `from_quads`.